flag_engine/api/schemas.py

```diff
@@ -130,6 +130,7 @@ class DjangoProjectSchema(BaseProjectSchema):
     segments = DjangoRelatedManagerField(
         fields.Nested(DjangoSegmentSchema), required=False, dump_only=True
     )
+    local_only_feature_ids = fields.List(fields.Int(), required=False, dump_only=True)
```

Comment:
I am not sure if `local_only` is the correct name here, since SDK's not running in local Eval mode (with server side key) can still access these flags

Reply:
Agreed, switched to more specific `server_key_only`.

Reply:
I think the other way around should give us a small array? i.e: something like `client_side_features_ids` because every feature by default is `server side only` ?

Reply:
My understanding was that current behaviour (server+client) should be the default one.